game/src/js/scenes/DemoScene.js

```diff
@@ -41,7 +41,16 @@ export default class DemoScene extends BaseScene {
     this.golemMonster = new GolemMonster(this, -300, -100);
     this.knight = new Knight(this, -100, 100);
 
-    this.cameras.main.startFollow(this.priest);
+    //movePlayer stuff IE impactSprite
```

Comment:
It is preferred to use the existing `Character` class that every individual character subclasses. Remember that we will be adding motion to multiple characters at once.

See lines 37-42.